src/instana/util/traceutils.py

```diff
@@ -104,3 +104,13 @@ def is_service_or_endpoint_ignored(
         service.lower() in agent.options.ignore_endpoints
         or f"{service.lower()}.{endpoint.lower()}" in agent.options.ignore_endpoints
     )
+
+
+def get_operation_specifier(span_name: str) -> str:
+    """Get the specific operation specifier for the given span."""
+    operation_specifier = ""
```

Comment:
I suggest you return `Optional[str]` and handle the `None` return when calling this function.
```suggestion
def get_operation_specifier(span_name: str) -> Optional[str]:
    """Get the specific operation specifier for the given span name."""
    operation_specifier = None
```